packages/frontend/src/components/EthreumCell.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+
+interface EthereumCellProps {
+  project: {
+    slug: string
+    name: string
+  }
+}
+
+export function EthereumCell(props: EthereumCellProps) {
+  return (
+    <>
+      <div className="ProjectCell">
+        <img
+          className="ProjectCell-Icon"
+          src={`/icons/${props.project.slug}.png`}
+          alt={`${props.project.name} logo`}
+        />
+        {props.project.name}
+      </div>
```

Comment:
Please style all new components using tailwind

Reply:
Oooh, ok, so the SCSS is just a leftover from old days?

Reply:
Ethereum cell is sharing all the styling with ProjectLink tho, excluding the hover effect, looks to me like a good idea to use SCSS to keep it consistent here.

Reply:
> Ethereum cell is sharing all the styling with ProjectLink tho, excluding the hover effect, looks to me like a good idea to use SCSS to keep it consistent here.

I'll let it slide now, but I'm going to definitely enforce it more after I implement the new designs and reduce SCSS use even more